kotlinlib/src/mill/kotlinlib/ktlint/KtlintModule.scala

```diff
@@ -74,15 +81,14 @@ object KtlintModule extends ExternalModule with KtlintModule with TaskModule {
 
   /**
    * Checks the Kotlin source files formatting without reformatting the files.
-   *
-   * @param check if an exception should be raised when formatting errors are found
-   *              - when set, files are not formatted
    */
   def checkFormatAll(
-      check: mainargs.Flag = mainargs.Flag(value = false)
+      @arg(positional = true) sources: Tasks[Seq[PathRef]] =
+        Tasks.resolveMainDefault("__.sources")
   ): Command[Unit] = Task.Command {
     ktlintAction(
-      KtlintArgs(format = false, check = check.value),
+      KtlintArgs(format = false, check = false),
```

Comment:
Just a guess, but isn't this a typo? Shouldn't this be
```scala
KtlintArgs(format = false, check = true),
```

Reply:
good catch, it is a copy-paste typo here. https://github.com/com-lihaoyi/mill/pull/4247